README-RU.md

```diff
@@ -0,0 +1,113 @@
+[![Pub](https://img.shields.io/pub/v/flame.svg?style=popout)](https://pub.dartlang.org/packages/flame) ![Test](https://github.com/flame-engine/flame/workflows/Test/badge.svg?branch=master&event=push) [![Discord](https://img.shields.io/discord/509714518008528896.svg)](https://discord.gg/pxrBmy4)
+
+<img src="https://i.imgur.com/vFDilXT.png" width="400">
+
+## О версии 1.0.0
+
+Это релиз-кандидат версии 1.0.0 движка Flame. Эта версия не готова к использованию в продакшене, часть когда из документации не адаптирована под нее, кроме того, многое еще может поменяться перед релизом.
+
+Используйте эту версию если хотите посмотреть каким будет Flutter и оставить обратную связь по структуре или предложить новые фичи.
```

Comment:
Shouldn't this say "Flame" and not "Flutter"?

Reply:
sure!